restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -173,6 +176,73 @@ public Response postDoc(
         });
   }
 
+  @POST
+  @ManagedAsync
+  @ApiOperation(
+      value = "Write multiple documents in one request",
+      notes =
+          "Auto-generates an ID for the newly created document if an idPath is not provided as a query parameter. When an idPath is provided, this operation is idempotent.",
```

Comment:
> this operation is idempotent

As in a NOOP or it's an upsert?

Reply:
It will be an upsert - if you provide an `idPath` and do two back-to-back requests, the second request will update instead of inserting